.github/CODEOWNERS

```diff
@@ -2022,7 +2019,6 @@ spec/support/schemas/all_triage_teams.json @department-of-veterans-affairs/vfs-m
 spec/support/schemas/upload_supporting_evidence.json @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/support/schemas/user_loa1.json @department-of-veterans-affairs/octo-identity
 spec/support/schemas/user_loa3.json @department-of-veterans-affairs/octo-identity
-spec/support/schemas/va_forms @department-of-veterans-affairs/platform-va-product-forms @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
I confirmed that these aren't the schemas that are used by the Sitewide Public Websites team to confirm that the VA Forms API response conforms to the schema. Their file is stored under `lib/forms/schemas/forms.json`.